src/Uri.php

```diff
@@ -690,7 +701,13 @@ private function validateState()
                 throw new \InvalidArgumentException('A relative URI must not have a path beginning with a segment containing a colon');
             }
         } elseif (isset($this->path[0]) && $this->path[0] !== '/') {
-            throw new \InvalidArgumentException('The path of a URI with an authority must start with a slash "/" or be empty');
+            @trigger_error(
```

Comment:
I would like also to notice that if you do not add the mute there will be developers that their tests might fail. 

Although in some cases if they had tests they would have already made the necessary changes.